packages/launcher-component/src/core/data-loader/data-loader.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { useEffect, useState } from 'react';
 import { InProgressIcon } from '@patternfly/react-icons';
+import { AlertError } from '../hub-n-spoke/index';
```

Comment:
aw snap, I haven't seen `AlertError` was imported from `hub-n-spoke` let's move it too `stuff` wdyt?

Reply:
wait why do we need the loading error when it here as well, wouldn't this already work?

Reply:
@edewit the `AlertError` in `hub-n-spoke` is used for launch errors only.